app/extensions/flask_sqlalchemy/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""
+Flask-SQLAlchemy adapter
+------------------------
+"""
+import sqlite3
+
+from sqlalchemy import engine, MetaData
+
+from flask_sqlalchemy import SQLAlchemy as BaseSQLAlchemy
+
+
+def set_sqlite_pragma(dbapi_connection, connection_record):
+    # pylint: disable=unused-argument
```

Comment:
Why? Is connection_record needed ?

Reply:
It is required by SQLAlchemy when it instatiates the DB connection.